example/example.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/knei-knurow/frames"
+)
+
+func main() {
```

Comment:
This example is not really a good example, but it can be left here for now:) We can always improve it in the future.